Kernel/Process.cpp

```diff
@@ -820,6 +813,17 @@ void Process::unblock_waiters(Thread::WaitBlocker::UnblockFlags flags, u8 signal
         waiter_process->m_wait_blocker_set.unblock(*this, flags, signal);
 }
 
+void Process::remove_from_secondary_lists()
```

Comment:
why not name it explicitly, using a name that would describe what is being done at this very moment? (i.e. removing from jail process list)

Reply:
This is called here so I rather keep this way:
https://github.com/SerenityOS/serenity/blob/2b6c44852c6cb8127d870ec72b5d4c82df7825e0/Kernel/Library/ListedRefCounted.h#L37-L39